### DISCUSSION TITLE
Is it possible to explore RAVE without an understanding of GitHub?

### DISCUSSION THREAD

**Discussion by ign3ousr0ck** [acids-ircam/RAVE#207] [General]
Hello developers of RAVE!

I'm a 29 y/o composer who is working on a commission for an AI generated dance piece. I have taken some university courses in Max/MSP 10 years ago. I would like to explore this super fascinating device. Alas, I am embarrassed that I cannot even open RAVE in Max, because I don't understand GitHub. I have literally spent an hour trying to figure out the installation instruction 'Install RAVE with: pip install acids-rave'. What does this mean? I am watching GitHub tutorial videos for an hour and am no closer to understanding how to take my first step with this device.

 Is further instruction possible in the supporting documentation, or should musicians with no understanding of GitHub just give up?

**Comment by caillonantoine** [comment]
Hey :) training new rave models does require a bit of command line usage at the moment. I'm working on a hands on video serie to help people understand the installation / preprocessing / training / export / usage of rave, from beginning to end. For now, you might want to experiment with nn~ and some pretrained models !

**Reply by caillonantoine** [reply]
Nn~ https://github.com/acids-ircam/nn_tilde

**Reply by caillonantoine** [reply]
Pretrained models https://acids-ircam.github.io/rave_models_download

**Reply by caillonantoine** [reply]
I have added the link to the rave discord on the readme, if you want to have a chat about it :)

**Comment by ign3ousr0ck** [comment]
Thanks so much for the generous response, Antoine. I eventually figured out command line, but then struggled for hours with pip until I figured out I needed to run 'pip3 install acids-rave'. A 5 minute tutorial video would definitely be super helpful for basic users like me! I've been generating cool pieces with Dance Diffusion using Colab notebooks for the last couple weeks, so users like me can get creative with AI, with maybe a bit more help :)

Been having fun with the pretrained nn~ models, thanks so much.

Would love to try and train my own model, the discord link in the readme doesn't open up your server for me though. Thanks again for everything and hope to continue exploring your beautiful work!
